addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -470,19 +470,17 @@
                     </div>
                 </div>
 
-                <p t-if="o.move_type in ('out_invoice', 'in_refund') and o.payment_reference" name="payment_communication">
-                    <div class="row">
-                        <div class="col-2 offset-6 text-nowrap">
-                            <strong>Payment Reference:</strong>
-                        </div>
-                        <div class="col-2 text-nowrap">
-                            <span class="fw-bold" t-field="o.payment_reference"/>
-                        </div>
-                        <div class="col-2 text-end">
-                            <strong>:رقم إشارة الدفعة</strong>
-                        </div>
+                <div class="row" t-if="o.move_type in ('out_invoice', 'in_refund') and o.payment_reference" name="payment_communication">
+                    <div class="col-2 offset-6">
+                        <strong>Payment Reference:</strong>
                     </div>
-                </p>
+                    <div class="col-2">
+                        <span class="fw-bold" t-field="o.payment_reference"/>
+                    </div>
+                    <div class="col-2 text-end">
+                        <strong style="white-space:nowrap">:رقم إشارة الدفعة</strong>
```

Comment:
maybe you should use equivalent `class="text-nowrap"` instead, but in other place of the template we also use direct style so both seems ok